src/util/sortOrders.js

```diff
@@ -4,23 +4,27 @@
  */
 
 const byName = (a, b) => a.name.localeCompare(b.name, OC.getLanguage(), { numeric: true })
-const byTitle = (a, b) => a.title.localeCompare(b.title, OC.getLanguage(), { numeric: true })
-const byTimestamp = (a, b) => b.timestamp - a.timestamp
+const byTitleAsc = (a, b) => a.title.localeCompare(b.title, OC.getLanguage(), { numeric: true })
+const byTitleDesc = (a, b) => b.title.localeCompare(a.title, OC.getLanguage(), { numeric: true })
```

Comment:
Maybe easier to understand / parse when expressed in terms of the ascending option?

```suggestion
const byTitleDesc = (a, b) => byTitleAsc(b, a)
```

Reply:
Great idea, changed!